modules/UI/videolayout/RemoteVideo.js

```diff
@@ -46,8 +46,7 @@ function createContainer(spanId) {
         <div class = 'videocontainer__hoverOverlay'></div>
         <div class = 'displayNameContainer'></div>
         <div class = 'avatar-container'></div>
-        <div class ='presence-label-container'></div>
-        <span class = 'remotevideomenu'></span>`;
+        <div class ='presence-label-container'></div>`;
```

Comment:
There's not a chance to disable that functionality by config?

Reply:
Unfortunately no :/  config right now has very limited attributes we could play around, but they are enriching the code day by day. Btw, we should take care when we merge the origin repo because they are introducing new features and we have to take care of that and not get surprised when our OVS looks totally different lol :)